source/1-SDLC-organization/lib/cdk-bootstrap-template.yml

```diff
@@ -0,0 +1,751 @@
+                  - kms:Decrypt
+                  - kms:DescribeKey
+                  - kms:Encrypt
+                  - kms:ReEncrypt*
+                  - kms:GenerateDataKey*
+                Resource: "*"
+                Effect: Allow
+              - Action: iam:PassRole
+                Resource:
+                  Fn::Sub: "${CloudFormationExecutionRole.Arn}"
+                Effect: Allow
+              - Sid: CliPermissions
+                Action:
+                  # Permissions needed by the CLI when doing `cdk deploy`.
+                  # Our CI/CD does not need DeleteStack,
+                  # but we also want to use this role from the CLI,
+                  # and there you can call `cdk destroy`
+                  - cloudformation:DescribeStackEvents
+                  - cloudformation:GetTemplate
+                  - cloudformation:DeleteStack
+                  - cloudformation:UpdateTerminationProtection
+                  - sts:GetCallerIdentity
+                Resource: "*"
+                Effect: Allow
+            Version: '2012-10-17'
+          PolicyName: default
+      RoleName:
+        Fn::Sub: cdk-${Qualifier}-deploy-role-${AWS::AccountId}-${AWS::Region}
+  CloudFormationExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service: cloudformation.amazonaws.com
+        Version: '2012-10-17'
+      ManagedPolicyArns:
+        Fn::If:
+          - HasCloudFormationExecutionPolicies
+          - Ref: CloudFormationExecutionPolicies
+          - Fn::If:
+              - HasTrustedAccounts
+              # The CLI will prevent this case from occurring
+              - Ref: AWS::NoValue
+              # The CLI will advertise that we picked this implicitly
+              - - Fn::Sub: "arn:${AWS::Partition}:iam::aws:policy/AdministratorAccess"
+      PermissionsBoundary:
+        Ref: CloudFormationExecutionPermissionsBoundary
+      RoleName:
+        Fn::Sub: cdk-${Qualifier}-cfn-exec-role-${AWS::AccountId}-${AWS::Region}
+  CloudFormationExecutionPermissionsBoundary:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action:
+              - s3:CreateBucket
+              - s3:DeleteBucket
+              - s3:DeleteBucketPolicy
+              - s3:DeleteJobTagging
+              - s3:DeleteObjectTagging
+              - s3:DeleteObjectVersionTagging
+              - s3:PutBucketPolicy
+              - s3:PutBucketPublicAccessBlock
+              - s3:PutBucketTagging
+              - s3:PutJobTagging
+              - s3:PutObjectTagging
+              - s3:PutObjectVersionTagging
+              - s3:GetEncryptionConfiguration
+              - s3:PutEncryptionConfiguration
+            Resource:
+              - "*"
```

Comment:
`cdk-pipeline` creates both a S3 bucket and 6 IAM Roles. I tried to find pattern of them to limit resource list, but all of them can be changed if the stack name is long. (stack name is used as a prefix). So I have to use * here